.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+<!-- Please use this template when creating an issue. 
+- Please check the boxes after you've created your issue.
+- Please use the latest version of
+    ACF Content Analysis for Yoast SEO,
+    Yoast SEO
+    and Advanced Custom Fields.
+-->
+
+* [ ] I've read and understood the [contribution guidelines](https://github.com/Yoast/yoast-acf-analysis/blob/develop/.github/CONTRIBUTING.md).
+* [ ] I've searched for any related issues and avoided creating a duplicate issue.
+
+### Please give us a description of what happened.
+
+
+
+
+### Please describe what you expected to happen and why.
+
+
+
+
+### How can we reproduce this behavior?
+1. 
+2. 
+3. 
+
+### Technical info
+* WordPress version: 
+* ACF Content Analysis for Yoast SEO version: 
+* Yoast SEO version: 
+* ACF version: 
```

Comment:
The free version will soon move to 5.* too so whether it is free or pro would be nice to mention. Maybe:

> ACF version (including if Free or Pro):